index.js

```diff
@@ -333,7 +333,7 @@ let Fixtures = {
             let getRequiredFieldPromise = MetadataHandler.getRequiredFields(model.module)
             .then((requiredFields) => {
                 _.each(requiredFields, (field) => {
-                    if (!request.data[field.name]) {
+                    if (_.isUndefined(request.data[field.name])) {
```

Comment:
We should only try to automagically fill required fields if they haven't been supplied.